docs/README.md

```diff
@@ -71,11 +71,15 @@ Instructions for running and initializing Keycloak locally:
   You can access keycloak at http://localhost:8080
 
   Keycloak ADMIN credentials:
-  # Create Realms
```

Comment:
This was shown as a large heading... decided to throw the correction in here since I was already touching the readme